src/windows/system.rs

```diff
@@ -63,7 +63,14 @@ unsafe impl<T> Sync for Wrap<T> {}
 
 unsafe fn boot_time() -> u64 {
     match SystemTime::now().duration_since(SystemTime::UNIX_EPOCH) {
-        Ok(n) => n.as_secs().saturating_sub(GetTickCount64() / 1_000),
+        Ok(n) => {
+            let system_time_ns = n.as_nanos();
```

Comment:
Please add your explanation here as a comment as future me will likely wonder what the hell is going on in here. 😆

Reply:
We were effectively rounding twice: `Duration::as_secs` discards nanosecond precision, and `GetTickCount64() / 1_000` truncates due to integer division. This leads to inconsistent seconds depending on how each value is rounded. 

By using nanoseconds for the subtraction and converting to seconds only once at the end, we achieve consistent rounding and stable results.

Hi future Guillaume! 👋

Reply:
No I meant as a code comment in the code. :stuck_out_tongue:

Reply:
I must admit I was slightly confused 🤣 I'll add a comment!